_posts/2020-08-06-changelog-128.markdown

```diff
@@ -0,0 +1,171 @@
+    
+* Macro expansion fixes:    
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5862 -->
+    * Fix `ArrayIndexOutOfBoundsException` during macro call arguments completion
+    
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5810 -->
+    * Replace `IntellijRustDollarCrate` with a crate name in `Show macro expansion` action
+    
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5796 -->
+    * Resolve `$crate` via hygiene
+    
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5855 -->
+    * Don't skip doc comments provided via macro definition
+    
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5777 -->
+    * Fix groups substitution in macro expansion
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5857 -->
+* Find and replace occurrences located outside a function in `Introduce constant/variable` refactoring (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5856 -->
+* Properly check spelling in lifetimes. Now `'static` is not reported as a typo
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5853 -->
+* Fix name resolution of private reexport with `crate::` path
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5849 -->
+* Include `RUSTC` in environment when executing Cargo (by [@tamird])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5842 -->
+* Don't show parameter hints for single-parameter closures
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5838 -->
+* Fix stdlib source paths on nightly. This fixes `Cargo project update failed` error.
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5822 -->
+* Don't provide color highlighting in batch mode. It prevents empty items in `Inspection Result` tool window
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5820 -->
+* Suppress `snake_case` naming lint for `#[no_mangle]` functions (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5819 -->
+* Add `unsafe` modifier to functions [extracted](https://www.jetbrains.com/help/idea/extract-method.html) from unsafe functions (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5801 -->
+* Fix navigation to derive items in 2020.2
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5799 -->
+* Use correct language name for Toml color settings configurable
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5789 -->
+* Don't create checkboxes in inlay hint settings manually
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5779 -->
+* Fix mod declarations with `#[path = "mod.rs"]` attribute
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5766 -->
+* Attach module without a parent to a parent module in `Attach file to module` fix (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5661 -->
+* Prefer trait methods before private inherent methods in completion (by [@Kobzol])
```

Comment:
Let's move this item higher since it's more important than, for example, https://github.com/intellij-rust/intellij-rust/pull/5799 or https://github.com/intellij-rust/intellij-rust/pull/5789